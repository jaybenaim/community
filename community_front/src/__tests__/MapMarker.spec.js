import React from "react";
import { create } from "react-test-renderer";

describe("Map Component should render markers", () => {
  test("MapMarkers are rendering onto the map", () => {
    const mapMarker = create(<div className="hammer"></div>);
    expect(mapMarker).toMatchSnapshot();
  });
});
