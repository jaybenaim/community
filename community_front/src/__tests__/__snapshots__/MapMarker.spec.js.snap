// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Component should render markers MapMarkers are rendering onto the map 1`] = `
<div
  className="hammer"
/>
`;
